secure_message/repository/retriever.py

```diff
@@ -280,6 +280,4 @@ def check_msg_id_is_a_draft(draft_id, user):
             logger.error('Error retrieving message from database', error=e)
             raise InternalServerError(description="Error retrieving message from database")
 
-        if result is None:
-            return False, result
-        return True, result.serialize(user)
+        return result.serialize(user) if result else None
```

Comment:
Why are you checking if result evaluates to True? If you move this in to the try block as:

```Python
return result.serialize(user)
```

it will still work, as result will always be an SqlAlchemy object or an exception will be thrown.

Reply:
Done

Reply:
Actually , not quite . The serializer blows when zero records are returned, and i Need to distinguish if a message id was a draft in the case that someone attempts a put on a message id 
that is not a draft